packages/smooth_app/lib/bottom_sheet_views/product_copy_view.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class CopyToListView extends StatelessWidget {
+  const CopyToListView(
+    this._scrollController, {
+    this.callback,
+    @required this.children,
+  });
+
+  final ScrollController _scrollController;
+  final Function callback;
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: Container(
+        height: MediaQuery.of(context).size.height * 0.9,
+        child: Stack(
```

Comment:
A `Stack` with only one child: please remove the `Stack`.

Reply:
I must have been out of sorts :)